File: VB/DXWebApplication1/Models/DataHelper.vb
Imports System
Imports System.Linq
Imports DevExpress.XtraRichEdit
Imports DXWebApplication1.Models.EF

Namespace DXWebApplication1.Models
	Public Class DataHelper
		Public Shared Function GetDocument() As Byte()
			Dim context As New DataClassesDataContext()
			Return context.Docs.FirstOrDefault().DocBytes.ToArray()
		End Function

		Public Shared Sub SaveDocument(ByVal bytes() As Byte)
			Dim context As New DataClassesDataContext()
			context.Docs.FirstOrDefault().DocBytes = bytes
			context.SaveChanges()
		End Sub
	End Class
	Public Class RichEditData
		Public Property DocumentId() As String
		Public Property DocumentFormat() As DocumentFormat
		Public Property Document() As Byte()
	End Class
End Namespace

File: CS/DXWebApplication1/Models/DataHelper.cs
using System;
using System.Linq;
using DevExpress.XtraRichEdit;
using DXWebApplication1.Models.EF;

namespace DXWebApplication1.Models {
    public class DataHelper {
        public static byte[] GetDocument() {
            DataClassesDataContext context = new DataClassesDataContext();
            return context.Docs.FirstOrDefault().DocBytes.ToArray();
        }

        public static void SaveDocument(byte[] bytes) {
            DataClassesDataContext context = new DataClassesDataContext();
            context.Docs.FirstOrDefault().DocBytes = bytes;
            context.SaveChanges();
        }
    }
    public class RichEditData {
        public string DocumentId { get; set; }
        public DocumentFormat DocumentFormat { get; set; }
        public byte[] Document { get; set; }
    }
}